tst/testlibgap/common.h

```diff
@@ -0,0 +1,11 @@
+#include <stdio.h>
+#include <unistd.h>
+#include <compiled.h>
```

Comment:
I'd move the `compiled.h` include to `common.c`; after all, you don't want to expose all GAP innards to all test files, but rather want to focus on testing libgap-api.h, no?

Reply:
well, there are functions in `compiled.h` used outside of `common.c`, e.g. `CollectBags`. Thus doing this move gives
```
tst/testlibgap/basic.c:9:5: warning: implicit declaration of function ‘CollectBags’ [-Wimplicit-function-declaration]
     CollectBags(0, 1);    // full GC
     ^~~~~~~~~~~
...
```
I'd leave this as it is.